### ISSUE TITLE
Unit tests

### ISSUE THREAD

**Issue by joetidee** [ehellman/react-breakpoints#14]
Do you have any plans to write unit tests to this package?

**Comment by ehellman** [comment]
Yes! I've actually got a local branch where I've set up the basics for Jest. I'm adding this to projects right away so that it will be prioritised. Thanks!

**Comment by ehellman** [comment]
@joetidee I see it as feedback, which is always welcome! :) #16 is the PR for this, going to keep working on it this week.

Thanks a lot, I plan on keeping it that way too! While having docs, comments and examples to make it easier for both a junior all the way up to a senior to pick it up and use, or contribute for that matter!

**Comment by ehellman** [comment]
@joetidee If you see this and already e-mailed me can you please re-send it. My DNS has been messed up and e-mails after november 8th has been bouncing.
